package.json

```diff
@@ -53,6 +53,8 @@
     "testdouble": "^1.6.0"
   },
   "dependencies": {
+    "broccoli": "^0.16.9",
```

Comment:
you might want to use https://github.com/ember-cli/broccoli-builder here instead